internal/version/version.go

```diff
@@ -0,0 +1,17 @@
+package version
+
+var (
+	// Version shows the current notation version, optionally with pre-release.
+	Version = "0.5.1"
```

Comment:
Is it just a placeholder? Do you need to manually change the version for each release?

Reply:
It is just a placeholder and is for local build. GitHub action will use the tag name for the version.